server/tasks/stream_inference.py

```diff
@@ -69,6 +94,8 @@ async def run_streaming(
     serializer="pickle",
     on_failure=parent_job_error_handler,
     task_time_limit=settings.task_time_limit_sec,
+    task_ignore_result=True,
```

Comment:
Hm this doesnt return anything but perhaps not setting this (default is False, so celery does store what the task returns) caused some sort of leak

Reply:
Yep that's what I was thinking, but it's stored in redis, so if anything wouldn't the leak be in that container not the worker container? But I set it just in case